module/VuFind/src/VuFind/Db/Entity/UserResource.php

```diff
@@ -38,60 +38,85 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="user_resource", indexes={@ORM\Index(name="list_id", columns={"list_id"}), @ORM\Index(name="resource_id", columns={"resource_id"}), @ORM\Index(name="user_id", columns={"user_id"})})
+ * @ORM\Table(name="user_resource",
+ *          indexes={@ORM\Index(name="list_id", columns={"list_id"}),
+ * @ORM\Index(name="resource_id",   columns={"resource_id"}),
+ * @ORM\Index(name="user_id",       columns={"user_id"})}
+ * )
  * @ORM\Entity
  */
 class UserResource implements EntityInterface
 {
     /**
+     * Unique ID.
+     *
      * @var int
      *
-     * @ORM\Column(name="id", type="integer", nullable=false)
+     * @ORM\Column(name="id",
+     *          type="integer",
+     *          nullable=false
+     * )
      * @ORM\Id
      * @ORM\GeneratedValue(strategy="IDENTITY")
      */
     protected $id;
 
     /**
+     * Notes associated with the resource.
+     *
      * @var string|null
      *
      * @ORM\Column(name="notes", type="text", length=65535, nullable=true)
      */
     protected $notes;
 
     /**
+     * Date saved.
+     *
      * @var \DateTime
      *
-     * @ORM\Column(name="saved", type="datetime", nullable=false, options={"default"="CURRENT_TIMESTAMP"})
+     * @ORM\Column(name="saved",
+     *          type="datetime",
+     *          nullable=false,
+     *          options={"default"="CURRENT_TIMESTAMP"})
      */
     protected $saved = 'CURRENT_TIMESTAMP';
 
     /**
+     * User id.
+     *
      * @var \VuFind\Db\Entity\User
      *
      * @ORM\ManyToOne(targetEntity="VuFind\Db\Entity\User")
      * @ORM\JoinColumns({
-     *   @ORM\JoinColumn(name="user_id", referencedColumnName="id")
+     * @ORM\JoinColumn(name="user_id",
+     *              referencedColumnName="id")
      * })
      */
     protected $user;
 
     /**
+     * Resource.
+     *
      * @var \VuFind\Db\Entity\Resource
      *
      * @ORM\ManyToOne(targetEntity="VuFind\Db\Entity\Resource")
      * @ORM\JoinColumns({
-     *   @ORM\JoinColumn(name="resource_id", referencedColumnName="id")
+     * @ORM\JoinColumn(name="resource_id",
+     *              referencedColumnName="id")
      * })
      */
     protected $resource;
 
     /**
+     * Id from userlist.
```

Comment:
Maybe change this to `User list ID` for consistency with the way other foreign keys have been described.